mlflow/utils/process.py

```diff
@@ -56,21 +62,31 @@ def _exec_cmd(
         raise ValueError("`extra_env` and `env` cannot be used at the same time")
 
     env = env if extra_env is None else {**os.environ, **extra_env}
-    prc = subprocess.run(
-        # In Python < 3.8, `subprocess.Popen` doesn't accpet a command containing path-like
-        # objects (e.g. `["ls", pathlib.Path("abc")]`) on Windows. To avoid this issue,
-        # stringify all elements in `cmd`. Note `str(pathlib.Path("abc"))` returns 'abc'.
-        map(str, cmd),
-        env=env,
-        check=False,
-        capture_output=capture_output,
-        text=True,
-        **kwargs,
-    )
-
-    if throw_on_error and prc.returncode != 0:
-        raise ShellCommandException.from_completed_process(prc)
-    return prc
+
+    # In Python < 3.8, `subprocess.Popen` doesn't accpet a command containing path-like
+    # objects (e.g. `["ls", pathlib.Path("abc")]`) on Windows. To avoid this issue,
+    # stringify all elements in `cmd`. Note `str(pathlib.Path("abc"))` returns 'abc'.
+    cmd = list(map(str, cmd))
```

Comment:
`ShellCommandException.from_completed_process` print log includes the "cmd", the `map` object cmd is not readable.

Reply:
Nice!